pavelib/utils/test/suites/nose_suite.py

```diff
@@ -120,7 +120,7 @@ def __enter__(self):
     @property
     def cmd(self):
         cmd = (
-            './manage.py {system} test --verbosity={verbosity} '
+            './manage.py {system} --contracts test --verbosity={verbosity} '
```

Comment:
We're adding it here so that our `manage.py` code doesn't call `contracts.disable_all()` before triggering the tests.